packages/ds-healthcare-gov/src/components/ThirdPartyExternalLink/index.ts

```diff
@@ -0,0 +1 @@
+import './ThirdPartyExternalLink';
```

Comment:
I'll write a ticket to go through and clean up the exports for component overrides and make them consistent across the `ds-healthcare-gov` and `ds-medicare-gov` packages. This works for now 👍